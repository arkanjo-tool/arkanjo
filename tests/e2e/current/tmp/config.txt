path of the current preprocess: tests/e2e/codebase
Finished time: Sat Feb 22 16:41:33 2025

