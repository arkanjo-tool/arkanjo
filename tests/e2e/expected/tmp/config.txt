path of the current preprocess: /home/luan/me/arkanjo/arkanjo/tests/e2e/codebase
Finished time: Sat Feb 22 16:17:46 2025

